### ISSUE TITLE
Use cascade operator in generated code 

### ISSUE THREAD

**Issue by kennethnym** [vanlooverenkoen/kiwi#70]
Currently the generated code produces something like this:
```
final KiwiContainer container = KiwiContainer();
container.registerSingleton();
container.registerSingleton();
container.registerSingleton();
container.registerSingleton();
```
Instead of repeating `container` over and over, the cascade operator can be used to chain method calls on `container`:
```
container
  ..registerSingleton()
  ..registerSingleton()
  ..registerSingleton()
```

**Comment by kennethnym** [comment]
i am more than happy to make a PR for this issue

**Comment by vanlooverenkoen** [comment]
That would be nice indeed. I am currently not having a lot of time to implement this. But I will review your pull requests if you could implement this.

**Comment by kennethnym** [comment]
Fixed in #72
